examples/src/main/java/org/apache/spark/examples/ml/JavaTokenizerExample.java

```diff
@@ -35,13 +35,11 @@
 import org.apache.spark.sql.types.Metadata;
 import org.apache.spark.sql.types.StructField;
 import org.apache.spark.sql.types.StructType;
-// $example off$
 
-// $example on:untyped_ops$
 // col("...") is preferable to df.col("...")
 import static org.apache.spark.sql.functions.callUDF;
 import static org.apache.spark.sql.functions.col;
-// $example off:untyped_ops$
+// $example off
```

Comment:
PS I actually fixed this in my big "Java 8" PR for SPARK-19550 because I needed to get the examples to compile. If we don't get that merged soon yes let's fix this in a quick PR